.ci/setup.sh

```diff
@@ -67,6 +67,11 @@ bash -f "${cidir}/install_kubernetes.sh"
 echo "Install Openshift"
 bash -f "${cidir}/install_openshift.sh"
 
+echo "Disable systemd-journald rate limit"
+sudo bash -c "echo RateLimitInterval=0s >> /etc/systemd/journald.conf"
+sudo bash -c "echo RateLimitBurst=0 >> /etc/systemd/journald.conf"
```

Comment:
looks good, but would this work if some predefined entries were already there ? Would this override them because they're located at the end ?

Reply:
ohhh, good question.
I've tried and the service at least is restarted successfully, but I'll better check to see if the entry is already there.